### ISSUE TITLE
Gradle build failed.   linesdk:5.1.1

### ISSUE THREAD

**Issue by app-chain-hagio** [line/line-sdk-unity#51]
pelase help me. I was able to build without problems until yesterday.

* What went wrong:
[2023-06-22T21:00:41Z - Unity] Could not determine the dependencies of task ':launcher:collectReleaseDependencies'.
[2023-06-22T21:00:41Z - Unity] > Could not resolve all task dependencies for configuration ':launcher:releaseRuntimeClasspath'.
[2023-06-22T21:00:41Z - Unity]    > Could not find com.linecorp:linesdk:5.1.1.
[2023-06-22T21:00:41Z - Unity]      Searched in the following locations:
[2023-06-22T21:00:41Z - Unity]        - https://dl.google.com/dl/android/maven2/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]        - https://jcenter.bintray.com/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]        - file:/BUILD_PATH/pXXXXXXXXXX-release-android/unityFiles/PIA/Library/Bee/Android/Prj/IL2CPP/Gradle/unityLibrary/libs/linesdk-5.1.1.jar
[2023-06-22T21:00:41Z - Unity]        - file:/BUILD_PATH/XXXXXXXX-release-android/unityFiles/PIA/Library/Bee/Android/Prj/IL2CPP/Gradle/unityLibrary/libs/linesdk.jar
[2023-06-22T21:00:41Z - Unity]        - https://maven.google.com/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]        - file:/BUILD_PATH/XXXXXXXXX-release-android/unityFiles/PIA/Assets/GeneratedLocalRepo/Firebase/m2repository/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]        - file:/BUILD_PATH/.m2/repository/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]        - https://repo.maven.apache.org/maven2/com/linecorp/linesdk/5.1.1/linesdk-5.1.1.pom
[2023-06-22T21:00:41Z - Unity]      Required by:
[2023-06-22T21:00:41Z - Unity]          project :launcher > project :unityLibrary

**Comment by onevcat** [comment]
@plateaukao I guess this is a similar case as https://github.com/line/flutter_line_sdk/issues/78

Can you also check and give it a fix? Thanks!

**Comment by onevcat** [comment]
Version 1.2.2 was released and should contain the fix for this issue. Please try to upgrade from [the release page](https://github.com/line/line-sdk-unity/releases/tag/1.2.2).

**Comment by shunsuke-yabuhana0coly-info** [comment]
@onevcat 
I am having the same problem.
I have updated to Version 1.2.2, but the 'line-sdk-unity-wrapper-release.aar' has not been updated, so I am still getting the same error in Unity as I did before the fix.

Also, when I try to build 'line-sdk-unity-wrapper' myself using AndroidStudio, I get the error 'Failed to resolve: com.linecorp:linesdk:5.8.1'.

I checked https://repo.maven.apache.org/maven2/com/linecorp/ and the path is 'com/linecorp/linesdk/linesdk/5.8.1'.
In mainTemplate.gradle, do I have to specify 'implementation com.linecorp.linesdk:linesdk:5.8.1'?
